apps/ledger-live-mobile/src/components/BackgroundGradient.tsx

```diff
@@ -16,9 +16,11 @@ import { useTheme } from "styled-components/native";
 function BackgroundGradient({
```

Comment:
I know it's not the purpose of this PR to create this component, but it would be good to protect the component with React.memo, there are a lot of svg components, it's costly to ask the native side to redraw every time.